project-keeper/src/main/java/com/exasol/projectkeeper/validators/changesfile/ChangesFileSection.java

```diff
@@ -11,7 +14,7 @@
  * </p>
  */
 public final class ChangesFileSection {
-
+    private static final Pattern FIXED_ISSUE_LINE_REGEXP = Pattern.compile("\\s*[-*]\\s*#(\\d+)\\s*:?\\s*(.+?)?\\s*");
```

Comment:
For improved readability, we could phrase the regex with simple spaces and after that replace them:
```suggestion
    private static final Pattern FIXED_ISSUE_LINE_REGEXP = Pattern.compile(" [-*] #(\\d+) :? (.+?)? ".replace(" ", "\\s*"));
```

Reply:
Smart!